src/com/lftechnology/java_training/jan12/WordsShuffle.java

```diff
@@ -0,0 +1,30 @@
+package com.lftechnology.java_training.jan12;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Write a program that uses Shuffle algorithm to print the words in its argument list in random order. Hint: Use List and Collection
+ * 
+ * @author Utsab Neupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class WordsShuffle {
+	private static final String[] WORDS_LIST = { "one", "two", "three", "four", "five", "six", "seven", "eight", "one" };
```

Comment:
You can directly initialize List using , 
`List<String> list = new ArrayList<String>(Arrays.asList("one", "two", "three", "four", "five", "six", "seven", "eight", "one"));

or yedi tehi na use garna cha bhane use
````
List<String> listOfWords = new ArrayList<>(Arrays.asList(word_one));
```
instead of 
```
for (String words : WORDS_LIST) {
 listOfWords.add(words);
 }